src/latex/agda-latex-macros.sty

```diff
@@ -3,6 +3,8 @@
 \RequirePackage{xspace}
 \RequirePackage{pifont}  %% needed for checkmark (\ding{51}).
 
+\newcommand\agdartdlang[1]{\href{https://agda.readthedocs.io/en/v2.6.4/language/#1}{agda.readthedocs.io\slash...\slash#1}\xspace}
```

Comment:
This looks broken in the PDF (I only know that the look is intended by reading this line). Can you make a proper bibliography that includes this instead?

Reply:
It doesn't seem broken to me.  Perhaps the way I had intended it to look appears broken to you.  I can change the style to conform to that of the link in the first section of the pdf.  I think a proper bibliography should be a new issue/PR since it involves touching other files as well.